council/contexts/_budget.py

```diff
@@ -125,7 +125,7 @@ def is_expired(self) -> bool:
         if self._deadline < time.monotonic():
             return True
 
-        return any(limit.value < 0.0 for limit in self._remaining)
+        return any(limit.value <= 0.0 for limit in self._remaining)
```

Comment:
@gkoch78 / @aflament I updated this line to ensure the budget is expired as soon as one of its consumption limits reach 0. Does that make sense to you?